core-bundle/src/Controller/ContentElement/VideoController.php

```diff
@@ -102,11 +109,11 @@ private function getVimeoSourceParameters(ContentModel $model): array
 
         return [
             'provider' => 'vimeo',
-            'video_id' => $videoId = $model->vimeo,
+            'video_id' => $videoId = explode('?', $model->vimeo)[0],
             'options' => $options,
-            'base_url' => $baseUrl = "https://player.vimeo.com/video/$videoId",
+            'base_url' => $baseUrl = "https://player.vimeo.com/video/$model->vimeo",
```

Comment:
```suggestion
            'base_url' => "https://player.vimeo.com/video/$model->vimeo",
```

Since the `$baseUrl` variable is not used anymore below.